### ISSUE TITLE
Docker image does not exist at liip/the-a11y-machine

### ISSUE THREAD

**Issue by louiscremen** [liip/TheA11yMachine#67]
Hey guys,

Still having issues with a previous issue submitted #66 , so I thought I would try it with the docker image
`$ docker run liip/the-a11y-machine --help
`

However, that failed:

```
 docker run liip/the-a11y-machine --help
Unable to find image 'liip/the-a11y-machine:latest' locally
Pulling repository docker.io/liip/the-a11y-machine
docker: Error: image liip/the-a11y-machine:latest not found.
```

So, after searching I found this:
[http://nickschuch.com/article/accessibility-as-a-service-with-kubernetes-and-the-a11y-machine/](http://nickschuch.com/article/accessibility-as-a-service-with-kubernetes-and-the-a11y-machine/)  and [https://github.com/liip/TheA11yMachine/pull/16#fork-destination-box](https://github.com/liip/TheA11yMachine/pull/16#fork-destination-box) which suggests to use 
`docker run nickschuch/the-a11y-machine` . This one works, and works without the issue I have in #66 . 

The next challenge I have is to make the docker image work for local files, which I think I can do with docker cp. I'll reply here if I get it to work.

**Comment by Hywan** [comment]
Hello :-),

Can you open a PR please?

**Comment by Hywan** [comment]
See the updated Installation Section: https://github.com/liip/TheA11yMachine#installation:

```sh
$ docker build -t liip/the-a11y-machine .
$ docker run liip/the-a11y-machine --help
```

This should fix your issue.

**Comment by esteinborn** [comment]
This only builds the image locally, it does not pull it from the docker hub, which was the original issue here.

The PR that was merged _should_ have set up auto-deployments to docker hub on each successful commit/tag. 

In order to successfully set this up, you'll need to follow the documentation on the docker hub, after creating an account there:
https://hub.docker.com/

then follow the instructions to "Create an automated build:"
Link your github account: https://hub.docker.com/account/authorized-services/github-permissions/

after linking, click "Create automated Build" and choose github, and choose TheA11yMachine.

Click create, and you will then be listed on the Docker Hub.

There may be a few more steps to make this happen. But it will stop other people from creating their own, old/outtaded versions of your great application:
https://hub.docker.com/r/nickschuch/the-a11y-machine/
https://hub.docker.com/r/lloydg/the-a11y-machine/
etc

Let me know if you'd like me to help you set this up.

**Comment by Hywan** [comment]
ping @jeanmonod
